client/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
```

Comment:
Please add a new section for `Installation` where we run the `yarn` command

Reply:
I've changed the installation from `yarn` to `npm` to make it consistent with the client. Reflected these in the README as well. Thanks!